qualification/__init__.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, cbf: CBFRemoteControl, stream_names: Sequence[str]) -> None:
         self.sync_time = cbf.sensors[f"{acv_name}.sync-time"].value
         self.scale_factor_timestamp = cbf.sensors[f"{acv_name}.scale-factor-timestamp"].value
         self.bandwidth = cbf.sensors[f"{acv_name}.bandwidth"].value
-        self.centre_freq = cbf.sensors[f"{acv_name}.centre-freq"].value
+        self.center_freq = cbf.sensors[f"{acv_name}.center-freq"].value
```

Comment:
The thought I had in mind was that everything internally would use "centre", except for the actual sensor which would be "center".

It's not a hill I'm willing to die on though.

Reply:
These variables are purely mirrors of the sensor values, so I thought it was best if they used exactly the same names as the sensors (modulo dash to underscore conversion).

Reply:
Fair point.